usaspending_api/references/abbreviations.py

```diff
@@ -0,0 +1,63 @@
+code_to_state = {
```

Comment:
I came across this library: https://github.com/unitedstates/python-us

Not sure if that would be a better solution compared to having our own abbreviations file, but something look at.